react-frontend/src/Home.js

```diff
@@ -1,16 +1,36 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import ButtonAppBar from './Navbar';
 
-const HomePage = () => (
-  <>
-    <ButtonAppBar />
-    <header>
-      <h1>My Simple React Home Page</h1>
-    </header>
-    <main>
-      <h1>Mission Log</h1>
-    </main>
-  </>
-);
+const HomePage = () => {
+  const [text, setText] = useState('');
+  const fullText = 'Welcome to Mission Log';
+
+  useEffect(() => {
+    if (text.length < fullText.length) {
+      setTimeout(() => setText(fullText.slice(0, text.length + 1)), 100);
+    }
+  }, [text]);
+
+  return (
+    <div style={{
+      minHeight: '100vh',
+      backgroundImage: 'radial-gradient(circle at center, #0a4700 0%, #002400 80%, #000000 100%)',
+    }}
+    >
+      <ButtonAppBar />
```

Comment:
This may be another feature, however since the navigation bar is part of the homepage it would be nice to style the navbar to dark green.

Reply:
Hey, Ishan! I do plan on changing the navbar so that it matches the homepage in the near future, thanks!